projects/scullyio/ng-lib/src/lib/idleMonitor/idle-monitor.service.ts

```diff
@@ -26,7 +27,7 @@ export class IdleMonitorService {
   private appReady = new Event('AngularReady', {bubbles: true, cancelable: false});
   private appTimeout = new Event('AngularTimeout', {bubbles: true, cancelable: false});
 
-  constructor(private zone: NgZone, private router: Router) {
+  constructor(private zone: NgZone, private router: Router, private tss: TransferStateService) {
```

Comment:
I'm injecting it here because we are starting the idle-service anyway. This will get refactored in an upcomming PR